.cirrus.yml

```diff
@@ -0,0 +1,49 @@
+task:
+  trigger_type: manual
+  name: E2E/Integration Tests
+  timeout_in: 45
+  container:
+    image: ubuntu:21.04
+    kvm: true # required for E2E/Integrations tests
+    cpu: 4
+    memory: 8G
+  env:
+    DEBIAN_FRONTEND: noninteractive
+    CI: true
+    FORCE_COLOR: 1 # force terminal color
+    CIRRUS_CLONE_DEPTH: 0 # Required to get app version
+    PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1 # disable auto-browser download
+    KEYRING: /usr/share/keyrings/nodesource.gpg
+    NODE_VERSION: node_14.x # set node version e.g: "node_16.x" for Node 16 LTS
+    DISTRO: hirsute # release codename for Ubuntu 21.04 - see "lsb_release -s -c"
+  install_deps_script: |
+    apt-get update
+    apt-get install -y --no-install-recommends ca-certificates gcc g++ curl git golang openssh-client make netcat sudo vim gpg \
+                    libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+    curl -sLf -o /dev/null "https://deb.nodesource.com/"$NODE_VERSION"/dists/"$DISTRO"/Release"
+    curl -s https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | tee "$KEYRING" >/dev/null
+    echo "deb [signed-by="$KEYRING"] https://deb.nodesource.com/"$NODE_VERSION" "$DISTRO" main" > /etc/apt/sources.list.d/nodesource.list
+    echo "deb-src [signed-by="$KEYRING"] https://deb.nodesource.com/"$NODE_VERSION" "$DISTRO" main" >> /etc/apt/sources.list.d/nodesource.list
+    apt-get update && apt-get install -y  nodejs && node -v && npm -v
+
+  # Tests should run as non-root user - setting up a test user
+  prepate_user_script:
+    - groupadd -g $(stat -c '%g' /dev/kvm) kvm
+    - useradd -m -G kvm ranchertest
+    - cp -rf . /home/ranchertest
+    - chown -R ranchertest.ranchertest /home/ranchertest
+
+  node_modules_cache:
+    folder: /home/ranchertest/node_modules
+    populate_script:
+      - sudo -iu ranchertest DEBUG=pw:install npm ci
+      - sudo -iu ranchertest ./node_modules/.bin/playwright install-deps
+
+  test_script:
+    - export KUBECONFIG=/home/ranchertest/.kube/config
+    - sudo -iu ranchertest CI=1 xvfb-run --auto-servernum -- npm run test:e2e
+
+  on_failure: 
+    custom_script: cp -R /home/ranchertest/e2e/reports/ . # custom_script workaround for cirrus bug: https://github.com/cirruslabs/cirrus-ci-agent/issues/197
+    logs_artifacts:
+      path: "/reports/*"
```

Comment:
Super-nit: new line at end of file.
(Does the path here need to be relative?)

Reply:
My linter didn't get the missing line :/ - thanks for catching

It does not to be relative so far. 
Cirrus runner will append/abstract `$CIRRUS_WORKING_DIR` on `path` string. Same as `/tmp/cirrus-ci-build/reports/*`

On the fix(Cirrus issue) it will require to pass the entire relative path.